src/Psalm/Internal/Cli/Psalm.php

```diff
@@ -904,6 +907,19 @@ private static function restart(array $options, int $threads): void
 
         // If Xdebug is enabled, restart without it
         $ini_handler->check();
+
+        if (PHP_MAJOR_VERSION < 8) {
+            fwrite(STDERR, PHP_EOL
+                . 'Run Psalm on PHP 8 to make use of JIT for a 20%+ performance boost!'
+                . PHP_EOL . PHP_EOL);
```

Comment:
IMO it's not very useful. If someone is running Psalm on PHP 7.x they probably have their reasons why they can't move to 8.x. Nagging them every time Psalm is run is not going to change that.